recipes/paragraph/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "paragraph" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5507561d8c5659047e16eb2e9cded1d9a4c1fc88489dd2bc09b76e660381c363
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - setuptools_scm
+    - attrs >=18.1.0
+
+test:
+  imports:
+    - paragraph
+
+
+about:
+  home: https://github.com/Othoz/paragraph
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

Comment:
The LICENSE file in the tarball is named `LICENSE`, so please add `license_file: LICENSE` instead.

Reply:
```suggestion
  license_file: LICENSE
```